common/testcontainers/testcontainers.go

```diff
@@ -106,6 +110,62 @@ func (t *TestcontainerApps) StartL2GethContainer() error {
 	return nil
 }
 
+// StartPoSL1Container starts the PoS L1 container by running the associated Docker Compose configuration
+func (t *TestcontainerApps) StartPoSL1Container() error {
+	var (
```

Comment:
How about remove the `StartL1GethContainer`? @colinlyguo

Reply:
make sense. let's remove `StartL1GethContainer`, using post-pos L1 instead.

Reply:
after the discussion, this will changed in next pr